### ISSUE TITLE
Catch GraphQL Errors

### ISSUE THREAD

**Issue by wodka** [ntegral/nestjs-sentry#16]
Currently the interceptor ignores errors if the context is "graphql"

https://github.com/ntegral/nestjs-sentry/blob/master/lib/services/sentry.interceptor.ts#L25

Something like this should work:

```ts
              case 'graphql':
                return this.captureGraphqlException(
                  scope, 
                  GqlExecutionContext.create(context), 
                  exception
                );
```

```ts
  private captureGraphqlException(scope: Scope, gqlContext: GqlExecutionContext, exception: any): void {
    const info = gqlContext.getInfo()
    const context = gqlContext.getContext() 

    scope.setExtra('type', info.parentType.name)

    if (context.req) {
      // req within graphql context needs modification in 
      const data = Handlers.parseRequest(<any>{} context.req, this.options);

      scope.setExtra('req', data.request);

      data.extra && scope.setExtras(data.extra);
      if (data.user) scope.setUser(data.user);
   }

    this.captureException(scope, exception);
  }
```

**Comment by davids-sparkshare** [comment]
Is this fixed with #23 ?

**Comment by ntegral** [comment]
This has been updated in the latest version.
